frontends/web/src/routes/account/send/send.tsx

```diff
@@ -379,7 +379,11 @@ class Send extends Component<Props, State> {
         this.setState({ proposedFee: undefined });
         if (errorCode) {
           this.unregisterEvents();
-          alertUser(errorCode, { callback: this.registerEvents });
+          if (errorCode === 'accountNotSynced') {
+            alertUser(this.props.t('send.error.accountNotSynced'), { callback: this.registerEvents });
```

Comment:
This is pretty ugly, but I guess that adapting the send page to calls that return instead of waiting if the account is not synced will require a bigger effort and a dedicated PR. 

Related to https://github.com/digitalbitbox/bitbox-wallet-app/pull/1859

Reply:
I am not sure you even need to handle the error in the frontend - while the sync is going on, the app shows a "snychronizing" screen and you cannot go to the send screen, right? Maybe simpler to just remove all of that. 

Btw, in my other recent PRs that do the same as this one (return errors instead of blocking) I didn't do any frontend error handling for the same reason.

Reply:
Yeah, that makes sense. I removed the frontend part, PTAL